packages/ibm-products/src/components/ConditionBuilder/utils/handleKeyboardEvents.js

```diff
@@ -0,0 +1,150 @@
+import { blockClass } from '../ConditionBuilderContext/DataConfigs';
+import { checkForHoldingKey } from './checkForHoldingKey';
+import { focusThisField, traverseClockVise, traverseReverse } from './util';
+
+export const handleKeyDown = (e, conditionBuilderRef) => {
```

Comment:
i think we should move the `handleKeyPressForPopover` declaration out of the function. also `e` to `evt`